Nos/Views/NoteButton.swift

```diff
@@ -15,20 +15,17 @@ import SwiftUI
 struct NoteButton: View {
     var note: Event
     var style = CardStyle.compact
+    var showFullMessage = false
 
     @EnvironmentObject private var router: Router
     @EnvironmentObject private var relayService: RelayService
 
     var body: some View {
         if let author = note.author {
             Button {
-                // If we are on the home feed or this is not the root note, allow clicks on replies
-                if router.path.count == 0 || note.eventReferences?.count ?? 0 > 0
-                    || router.navigationTitle == Localized.profile.rawValue {
-                    router.path.append(note)
-                }
```

Comment:
I think this logic was protecting us from being able to open duplicate thread views for the same root note

Reply:
will do in next PR.